CHANGELOG.md

```diff
@@ -69,6 +69,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Fixed a bug which dropped component updates on authority delegation.
 - The DeploymentLauncher checks the validity of the simulated players deployment name.
 - Worker configuration watcher only rebuilds worker configs when `*.worker.json` files are changed.
+- Fixed side-effecting tests.
```

Comment:
I would argue this is mostly an internal change and hence wouldn't need a release note, but I am bad at writing docs, so don't listen to me.

Reply:
would agree with that.